pages.fr/common/ls.md

```diff
@@ -0,0 +1,27 @@
+# ls
+
+> Liste le contenu d'un répertoire.
+
+- Affiche un fichier par ligne:
```

Comment:
What about:

    - Liste des fichiers un nom par ligne:

or even:

    - Liste des fichiers, un par ligne:

Sounds closer to the original, don't you think?